.changeset/perfect-bobcats-tickle.md

```diff
@@ -0,0 +1,6 @@
+---
+'@udecode/plate-common': patch
```

Comment:
Another changeset for this package may be nice

Reply:
What exactly do you mean?